### DISCUSSION TITLE
Nora Pay Method

### DISCUSSION THREAD

**Discussion by Marvin-VW** [andrei-tatar/node-red-contrib-smartnora#124] [Q&A]
Would it be possible to Pay without Credit Card? It would be much easier for me to pay with PayPal!

**Comment by andrei-tatar** [comment]
Unfortunately Stripe does not support paying with PayPal at this moment.

**Comment by Ixuser-1** [comment]
Would it be possible to make a donation via PayPal, as I have already done before for NORA? I would also pay twice the annual price for your additional effort.

**Reply by andrei-tatar** [reply]
Sorry, it's not possible to pay with PayPal at this moment. I see PayPal supports card payments:
https://www.paypal.com/us/digital-wallet/manage-money/paypal-debit-card

**Comment by andrei-tatar** [comment]
As of today, PayPal is a supported payment method, via Stripe and it's enabled for Smart NORA as well.
